### PR TITLE
Small updates to kraken

### PR DESCRIPTION
**Pull request by nekrut**

### CODE REVIEW

**Comment by bgruening** [comment]
@nekrut can you rebase this against https://github.com/galaxyproject/tools-devteam/pull/135
or the other way around.
ping @davebx

**Comment by nekrut** [comment]
Indeed @bgruening -> @davebx please rebase

File: tool_collections/kraken/kraken/kraken.xml [hunk 1]
```diff
@@ -9,7 +9,23 @@
     <command>
         <![CDATA[
         kraken --threads \${GALAXY_SLOTS:-1} @INPUT_DATABASE@ 
+        
+        #if isinstance( $input_sequences.datatype, $__app__.datatypes_registry.get_datatype_by_extension('fastq').__class__):
```
Comment (bgruening) [inline tool_collections/kraken/kraken/kraken.xml]: You are only allowing  'fastq, fasta' so it's probably easier to use $input_sequences.ext == 'fastq'

Comment (blankenberg) [inline tool_collections/kraken/kraken/kraken.xml]: allowing fastq will include all subclasses of fastq, which will have different ext, so this is correct. In the previous configuration, where it was specifying fastq and fastqsanger, the fastqsanger was superfluous.

Comment (blankenberg) [inline tool_collections/kraken/kraken/kraken.xml]: We should definitely have a much smoother why of performing this check, however.

Comment (bgruening) [inline tool_collections/kraken/kraken/kraken.xml]: I thought including 'fastq' would implicitly take all subclasses. Wouldn't this be a better way to handle this? I was just scared by this line ... a short cut would be nice :)

Comment (jmchilton) [inline tool_collections/kraken/kraken/kraken.xml]: Yeah - I think what you want is

```
$input_sequences.is_of_type( 'fastq' ) 
```

https://bitbucket.org/galaxy/galaxy-central/pull-request/457/allow-cheetah-tool-templates-to-reason/diff

Comment (bgruening) [inline tool_collections/kraken/kraken/kraken.xml]: Ha, I knew I had seen this before! Thanks a bunch John!

Comment (jmchilton) [inline tool_collections/kraken/kraken/kraken.xml]: Added docs for this https://github.com/galaxyproject/planemo/commit/d42cd7d77734c6b485b61d415ca4a3d60f01edf9

**Comment by bgruening** [comment]
Thanks everyone!

**Pull request merged**